Ready Makefile for debuild
 Ready Makefile for debuild by adding "install" and "uninstall" targets.
 .
 k3conf-0.1 (90-1) UNRELEASED; urgency=low
 .
   * Initial release. Closes: #nnnn
     <nnnn is the bug number of your ITP>
Author: root <>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2023-06-14

--- k3conf-0.1-90.orig/Makefile
+++ k3conf-0.1-90/Makefile
@@ -43,8 +43,6 @@ DEF_INC := $(shell $(CROSS_COMPILE)gcc
 
 DEF_INC_PATH = -I. -Iinclude
 
-STATIC_BUILD ?= -static
-
 MYCFLAGS+=$(CFLAGS) -c -Wall -Wextra -Wno-missing-field-initializers \
 		-I$(DEF_INC) $(DEF_INC_PATH) $(STATIC_BUILD)
 
@@ -55,6 +53,7 @@ MYCFLAGS+=-Os
 endif
 
 DESTDIR ?= target
+prefix = /usr/bin
 
 COMMONSOURCES=\
 		common/k3conf.c \
@@ -175,7 +174,7 @@ EXECUTABLE=	k3conf
 all: 		$(EXECUTABLE)
 
 $(EXECUTABLE):	$(ALLOBJECTS) builddate.o version.o
-		$(QUIET_CC) $(CC) $(STATIC_BUILD) $(LDFLAGS) $(ALLOBJECTS) builddate.o version.o\
+		$(QUIET_CC) $(CC) $(LDFLAGS) $(ALLOBJECTS) builddate.o version.o\
 		-lrt -o $@
 
 .c.o:
@@ -194,6 +193,12 @@ tags: $(ALLSOURCES)
 cscope: $(ALLSOURCES)
 	cscope -R -b
 
+install: $(EXECUTABLE)
+	install -D -m 0755 $(EXECUTABLE) $(DESTDIR)$(prefix)/k3conf
+
+uninstall:
+	-rm -f $(DESTDIR)$(prefix)/k3conf
+
 clean:
 		@echo "Cleaning up..."
 		-$(shell rm -f $(EXECUTABLE) *.o builddate.c version.c)
